src/components/Navbar/Navbar.js

```diff
@@ -1,30 +1,152 @@
-        <div className="row ml-auto">
+
+        <div className={`row ml-auto ${styles.desktopMenu}`}>
           <Link to="/">Home</Link>
           <div className="space-x-1"></div>
           <div className="space-x-1"></div>
-          <Link to="/categories">Categories</Link>
+          <Link to="/playlists">Playlists</Link>
+          <div className="space-x-1"></div>
+          <div className="space-x-1"></div>
+          <Link to="/account">Account</Link>
           <div className="space-x-1"></div>
           <div className="space-x-1"></div>
-          <Link to="">Playlists</Link>
+          <button
+            onClick={authBtnHandler}
+            className={`btn btn-solid-reverse btn-small`}
+          >
+            {user ? "Log Out" : "Log In"}
+          </button>
+          <button
+            onClick={switchTheme}
+            className={`btn btn-icon btn-ghost btn-small`}
+          >
+            <span className="material-icons-outlined">
+              {theme === "light" ? "dark_mode" : "light_mode"}
+            </span>
+          </button>
+        </div>
+        <div className={`${styles.mobileMenu} ml-auto`}>
+          <button
+            onClick={() => setShowMenuDrawer((val) => !val)}
+            className={`btn btn-icon btn-ghost btn-small ${styles.hamburgerIcon}`}
+          >
+            <span className="material-icons-outlined">menu</span>
+          </button>
+          <div
+            className={`${styles.menuDrawer} ${
+              showMenuDrawer && styles.menuDrawerActive
+            }`}
+            onClick={() => setShowMenuDrawer(false)}
+          >
+            <ul className={`flex flex-col items-center`}>
+              <li>
+                <Link className={`${styles.menuLink}`} to="/">
+                  Home
+                </Link>
+              </li>
+              <li>
+                <Link className={`${styles.menuLink}`} to="/playlists">
+                  Playlists
+                </Link>
+              </li>
+              <li>
+                <Link className={`${styles.menuLink}`} to="/account">
+                  Account
+                </Link>
+              </li>
+              <li>
+                <button
+                  onClick={switchTheme}
+                  className={`btn btn-icon btn-ghost btn-small ${styles.menuLink}`}
+                >
+                  <span className="material-icons-outlined">
+                    {theme === "light" ? "dark_mode" : "light_mode"}
+                  </span>
+                </button>
+              </li>
+              <li>
+                <button
+                  onClick={authBtnHandler}
+                  className={`btn btn-solid-reverse btn-small ${styles.menuLink}`}
+                >
+                  {user ? "Log Out" : "Log In"}
+                </button>
+              </li>
+            </ul>
+          </div>
         </div>
       </div>
     </div>
   );
 }
+
+const LogoutModal = ({ setShowLogoutModal }) => {
```

Comment:
This can be extracted out as a component, which will decrease code in this file.